src/app/exception_throw.py

```diff
@@ -0,0 +1,94 @@
+from fastapi import status
+from fastapi.responses import JSONResponse
+
+
+class exception_response:
+    status_code: int
+    status: str
+    message: str
+
+    def __init__(self, status_code: int, status: str, message: str):
+        self.status_code = status_code
+        self.status = status
+        self.message = message
+
+    def throw_er_with_json(self) -> JSONResponse:
+        return JSONResponse(
+            status_code=self.status_code,
+            content={"status": self.status, "message": self.message},
+        )
+
+
+def throw_500_global_exception() -> JSONResponse:
```

Comment:
How come you've defined an exception_response class and separate interceptor functions in the same file? I would separate these two concepts into different files, and probably wrap these throw functions inside an ExceptionInterceptor class.

Reply:
Refactored into two files